schema.json

```diff
@@ -745,6 +745,18 @@
     "name": "Carousel",
     "settings": [
       {
+        "type": "checkbox",

```

Comment:
How is this being used ? I do not see any corresponding template change to support it.